cpp/ps0006/ps0006_solv.cpp

```diff
@@ -149,3 +149,4 @@ void  counter(int end , int start  , int step )
         }
     }
 }
+// great work 😎 10/10 ⭐

```

Comment:
- [x] program is in the correct path
- [x] no syntax errors while compilation
- [x] no runtime errors, bugs or warnings while execution
- [x] formatted code to be clear and readable
- [x] commented some phrases for blocks of code
- [x] used `for` loop in a right way to to count up and down

you can check the [problem solution](https://onlinegdb.com/vdj-VXYv7) for more